### DISCUSSION TITLE
Will the 5.0 schemas be used for monitoring?

### DISCUSSION THREAD

**Discussion by Kate-Lyndegaard** [INSPIRE-MIF/helpdesk-validator#1065] [Q&A]
Dear @fabiovinci,

Will the updated schemas be used for monitoring?

Kind regards,
Kate

**Comment by fabiovinci** [comment]
Dear @Kate-Lyndegaard,

Could you please clarify your question?

Are you referring to the declaration of conformity in the dataset metadata?

**Comment by Kate-Lyndegaard** [comment]
Hi @fabiovinci,

Yes. Will the validator will be used in any way to automatically test dataset conformance of self-declared values? 

Kind regards,
Kate

**Comment by fabiovinci** [comment]
Hi @Kate-Lyndegaard,

according to the current rules, i.e. `Deprecated versions should no longer be used after 2 years after a major release`, version 4 can still be used. Please note that MIG/MIG-T will discuss these rules in the coming months; updates may follow.

**Comment by Kate-Lyndegaard** [comment]
Hi @fabiovinci,

Great. Thank you. I will stay tuned:-)
